### ISSUE TITLE
modify test runner to allow passing arguments

### ISSUE THREAD

**Issue by kevinburke** [balderdashy/sails-postgresql#129]
I'm trying to nail down a specific test, would be neat if `node tests/integration/runner.js` would pass arguments to mocha and respond to the `--help` flag. the `--grep` argument in particular would be nice.

**Comment by tjwebb** [comment]
You can pass arguments to mocha in test/integrations/runner.js. Not as convenient, but it is possible

**Comment by kevinburkeshyp** [comment]
any update here? running into this again

**Comment by sailsbot** [comment]
Thanks for posting, @kevinburke.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
